core/transport-options/src/grpc/file_system_master_options.proto

```diff
@@ -0,0 +1,172 @@
+syntax = "proto2";
```

Comment:
we should move to proto3 soon

Reply:
It's on my task list for the consolidation work after security.

Reply:
Looks like proto3 dropped support for checking presence of a value, so no more protoOption.hasQwe() family of methods. 
@madanadit , @aaudiber WDYT?